src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
> Can we consider reverting problematic commit instead? Seem less breaky.

But we don't even know what it's breaking. There's not enough utility to keep it, especially as an empty plain object.